uitools/widgets/Esri/ArcGISRuntime/Toolkit/BasemapGallery.cpp

```diff
@@ -0,0 +1,40 @@
+#include "BasemapGallery.h"
+#include "ui_BasemapGallery.h"
+
+
+//Toolkit controllers
+#include "BasemapGalleryController.h"
```

Comment:
newline here.

Reply:
you mean remove the newline?

Reply:
I meant to add one between lines 6 and 7. 🙂